src/narrowdown.h

```diff
@@ -23,6 +23,8 @@
 #include "zim_types.h"
 #include "debug.h"
 
+#include <zim/error.h>
+
```

Comment:
This change can be undone

Reply:
Indeed, I will remove the change on the whole file.